src/sass/components/_button.scss

```diff
@@ -1,36 +1,52 @@
-.btn-main {
-  background-color: #2a2a2a;
-  color: #ffffff;
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: 600;
-  padding: 10px 30px;
+.product-box {
```

Comment:
By nesting `.btn-main` inside `.product-box` you are breaking button inside `.post-info`. BTW We have the separate file dedicated to styles for `.product-box` - `_product-box.scss` - and IMO styles connected to `.product-box` should land there.